notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>Should we sample df after applying p_f_label and removing -1s or NaNs so we get 1000 meaningful records </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yes, we should remove all <code>-1</code>  as they aren't really usable since they represent all other outcome types as well as still running tests, right?</p>

Reply:
<p>Sure. I agree. Just that we were losing many values like that when we take the entire dataset. But yes, I will remove the NaN records instead.</p>

Reply:
<p>The dataset was originally around 21M values and now we have only 8M :/</p>

Reply:
8M should still be more then enough for this use case